app/src/main/java/org/simple/clinic/drugs/selectionv2/PrescribedDrugScreenV2.kt

```diff
@@ -122,4 +123,8 @@ class PrescribedDrugScreenV2(context: Context, attrs: AttributeSet) : LinearLayo
   fun showDosageSelectionSheet(drugName: String, patientUuid: UUID, prescribedDrugUuid: UUID?) {
     activity.startActivity(DosagePickerSheet.intent(context, drugName, patientUuid, prescribedDrugUuid))
   }
+
+  fun showUpdateCustomPrescriptionScreen(prescribedDrug: PrescribedDrug) {
+    activity.startActivity(CustomPrescriptionEntrySheetv2.intentForUpdatePrescription(context, prescribedDrug.uuid))
```

Comment:
What do you think about renaming `intentForUpdatePrescription()` -> `intentForUpdatingPrescription()`? Or `intentForPrescriptionUpdate`?

I don't have strong thoughts about this so it's okay if you want to skip.